### ISSUE TITLE
Could you publish the training loss curve?

### ISSUE THREAD

**Issue by 311dada** [SALT-NLP/LLaVAR#15]
Recently, I intend to reproduce your excellent work while meeting several training problems. So could you publish your training curve or final loss? Thanks a lot!

**Comment by StevenyzZhang** [comment]
![Screenshot 2023-10-17 at 10 07 52 AM](https://github.com/SALT-NLP/LLaVAR/assets/51267590/d6e464e8-3d70-4b53-ab9e-06665c57caef)
The loss should decrease pretty fast in the beginning and then slowly converge.

**Comment by 311dada** [comment]
Thanks for your help! We have reproduced the results!
